<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Egreso</class>
 <widget class="QMainWindow" name="Egreso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>243</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>HiCell-Egresos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/prefix2/icono.png</normaloff>:/new/prefix2/icono.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descripcion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_eDescripcion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_eTipo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Egresos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gastos</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Forma de Pago</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_eFormapago">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Efectivo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RedCompra</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_eGuardar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>356</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_eValor">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_eFecha">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>d-MM-yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Egresos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/hicell.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVenta">
    <property name="title">
     <string>Venta</string>
    </property>
    <addaction name="actionVenta"/>
    <addaction name="actionDetalle"/>
   </widget>
   <widget class="QMenu" name="menuInventario">
    <property name="title">
     <string>Inventario</string>
    </property>
    <addaction name="actionInventario"/>
    <addaction name="actionActualizar"/>
    <addaction name="actionAlertas"/>
   </widget>
   <widget class="QMenu" name="menuContabilidad">
    <property name="title">
     <string>Contabilidad</string>
    </property>
    <addaction name="actionContabilidad"/>
    <addaction name="actionEgresos"/>
   </widget>
   <widget class="QMenu" name="menuConfiguraci_n">
    <property name="title">
     <string>Configuraci√≥n</string>
    </property>
    <addaction name="actionCambio"/>
   </widget>
   <addaction name="menuVenta"/>
   <addaction name="menuInventario"/>
   <addaction name="menuContabilidad"/>
   <addaction name="menuConfiguraci_n"/>
  </widget>
  <action name="actionVenta">
   <property name="text">
    <string>Crear</string>
   </property>
  </action>
  <action name="actionActualizar">
   <property name="text">
    <string>Actualizar</string>
   </property>
  </action>
  <action name="actionContabilidad">
   <property name="text">
    <string>Visualizar</string>
   </property>
  </action>
  <action name="actionEgresos">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Agregar Egresos</string>
   </property>
  </action>
  <action name="actionInventario">
   <property name="text">
    <string>Visualizar</string>
   </property>
  </action>
  <action name="actionDetalle">
   <property name="text">
    <string>Modificar</string>
   </property>
  </action>
  <action name="actionCambio">
   <property name="text">
    <string>Cambio de Contrase√±a</string>
   </property>
  </action>
  <action name="actionAlertas">
   <property name="text">
    <string>Alertas</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
